src/main/sensors/boardalignment.c

```diff
@@ -40,8 +40,23 @@
 static bool standardBoardAlignment = true;     // board orientation correction
 static fp_rotationMatrix_t boardRotation;
 
-// no template required since defaults are zero
-PG_REGISTER(boardAlignment_t, boardAlignment, PG_BOARD_ALIGNMENT, 0);
+PG_REGISTER_WITH_RESET_TEMPLATE(boardAlignment_t, boardAlignment, PG_BOARD_ALIGNMENT, 1);
+
+#ifndef ALIGN_BOARD_ROLL
+#define ALIGN_BOARD_ROLL 0
+#endif
+#ifndef ALIGN_BOARD_PITCH
+#define ALIGN_BOARD_PITCH 0
+#endif
+#ifndef ALIGN_BOARD_YAW
```

Comment:
Can we please update these to DEFAULT_ as it indicates these are "default" settings, and updateable by the user. This is inline with the other default settings in config.h (though there is a mix currently). Be good to clean these up over time.

Reply:
Done.